### ISSUE TITLE
[hap2] hap2_zabbix_api can't get events & triggers from zabbix-2.0.3

### ISSUE THREAD

**Issue by ashie** [project-hatohol/hatohol#1799]
There are 2 problem:

* "state" isn't exist in trigger objects
    * https://www.zabbix.com/documentation/2.0/manual/appendix/api/trigger/definitions
* "hosts" isn't exist in event objects
    * https://www.zabbix.com/documentation/2.0/manual/appendix/api/event/definitions

**Comment by ashie** [comment]
> "state" isn't exist in trigger objects

Probably "value" is the property what we should use.
ArmZabbix uses "value", it doesn't use "state".

**Comment by ashie** [comment]
> "state" isn't exist in trigger objects

This property is available from 2.2:
https://www.zabbix.com/documentation/2.2/manual/api/reference/trigger/object

state:
```
(readonly) State of the trigger.

Possible values:
0 - (default) trigger state is up to date;
1 - current trigger state is unknown. 
```

value:
```
(readonly) Whether the trigger is in OK or problem state.

Possible values are:
0 - (default) OK;
1 - problem. 
```

**Comment by ashie** [comment]
> "hosts" isn't exist in event objects

In Zabbix 2.0 array of properties for "selectHosts" isn't supported:
https://www.zabbix.com/documentation/2.0/manual/appendix/api/event/get

selectHosts:
```
Return hosts containing the trigger that created the event in the hosts property.
Does not support shorten or arrays of properties. 
```

**Comment by ashie** [comment]
>     "state" isn't exist in trigger objects
> 
> Probably "value" is the property what we should use.
> ArmZabbix uses "value", it doesn't use "state".

It may be the cause of #1797.

When the "state" value is used, the value of fired triggers are always "0".

> 0 - (default) trigger state is up to date;
> 1 - current trigger state is unknown. 

"0" means "OK" in "value" property.

> 0 - (default) OK;
> 1 - problem. 

So the events concerned with the triggers aren't handled by actions which are filtered by
"problem" triggers.

**Comment by ashie** [comment]
> It may be the cause of #1797.

That's right. 
#1801 fixes also #1797 on my environment.
